src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -148,9 +148,13 @@ namespace AppInstaller::CLI::Workflow
         }
         else
         {
-            context.Reporter.Info() << Resource::String::MSStoreInstallOrUpdateFailed << ' ' << WINGET_OSTREAM_FORMAT_HRESULT(errorCode) << std::endl;
+            std::ostringstream ssError;
+            ssError << WINGET_OSTREAM_FORMAT_HRESULT(errorCode);
+            auto errorCodeString = Utility::LocIndString{ ssError.str() };

```

Comment:
These 3 lines can be in its own method that returns a LocIndString so that can be reused in other places that needs to print HResult